docs/books/learning_ansible/04-ansible-galaxy.md

```diff
@@ -41,7 +41,7 @@ ansible-galaxy role [import|init|install|login|remove|...]
 | `install`    | installs a role.                                          |
 | `remove`     | remove one or more roles.                                 |
 | `list`       | display the name and the version of installed roles.      |
-| `info`       | display information about a role.                        |
+| `info`       | display information about a role.                         |
 | `init`       | generate a skeleton of a new role.                        |
```

Comment:
While there is nothing wrong with this change, it isn't really necessary either. Personally, I prefer to line up all of the table elements as you are doing here, but the table should display fine w/o adding the space to make it line up with the others.